src/sqlite3gen.cpp

```diff
@@ -1110,172 +1872,201 @@ static void generateSqlite3ForMember(const MemberDef *md, const Definition *def)
   if (mdict!=0)
   {
     MemberSDict::IteratorDict mdi(*mdict);
-    MemberDef *rmd;
+    const MemberDef *rmd;
```

Comment:
This change (and others adding const in same pattern) are an attempt to parrot @groleo's change in #604. Won't pretend to fully understand; may not be correct.